en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configure-logs-adapter.md

```diff
@@ -0,0 +1,81 @@
+# Adapter Log Configurations
+
+The log configurations file for the adapter is located in [these directories]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configure-logs-overview/#log_config_toml) based on your Choreo Connect deployment method.
+
+The sample for Adapter log configurations looks similar to the following.
+
+``` toml
+# The logging configuration for Adapter
+
+## Adapter root Level configurations
+
+logfile = "logs/adapter.log"
+logLevel = "INFO"
+LogFormat = "TEXT"
+
+[rotation]
+MaxSize = 10    # In MegaBytes (MB)
+MaxBackups = 3
+MaxAge =  2   # In days
+Compress = true
+
+## Adapter package Level configurations
+
+[[pkg]]
+name = "github.com/wso2/product-microgateway/adapter/internal/adapter"
+logLevel = "INFO"
+
+[[pkg]]
+name = "github.com/wso2/product-microgateway/adapter/internal/oasparser"
+logLevel = "INFO"
+```
+
+## Adapter root Level configurations
+
+<table>
+    <thead>
+        <tr class="header">
+            <th>Property</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr class="odd">
+            <td><code>logfile</code></td>
+            <td>The file, which will be created inside the container for writting logs.</td>
+        </tr>
+        <tr class="even">
+            <td><code>logLevel</code></td>
+            <td>Upto which level, the logs should be printed. If this is set under root level, it will be applied through all packages, unless specified in package log levels. LogLevels can be <code>"DEBG"</code>, <code>"FATL"</code>, <code>"ERRO"</code>, <code>"WARN"</code>, <code>"INFO"</code>, <code>"PANC"</code></td>
```

Comment:
```suggestion
            <td>Upto which level the logs should be printed. If this is set under root level, it will be applied through all packages unless specified in package log levels. LogLevels can be <code>"DEBG"</code>, <code>"FATL"</code>, <code>"ERRO"</code>, <code>"WARN"</code>, <code>"INFO"</code>, <code>"PANC"</code></td>
```